### DISCUSSION TITLE
Weird things in WA676

### DISCUSSION THREAD

**Discussion by kevinwolz** [ncss-tech/soilDB#338] [Q&A] [answered]
Here's an AOI that contains a bit of soil survey area WA676 and adjacent areas:
![image](https://github.com/ncss-tech/soilDB/assets/11711566/088d8b96-a348-4d1e-85fc-21d7a77addab)

`sandtotal_r` from FY2023 gNATSGO
![image](https://github.com/ncss-tech/soilDB/assets/11711566/60924473-0a3f-439f-a214-cf3e21a88c90)

`sandtotal_r` from FY2024 gNATSGO
![image](https://github.com/ncss-tech/soilDB/assets/11711566/15a47927-1d05-4895-80c7-c0d3c0b5072f)

Here is the AOI as a KML zipped in a ZIP file:
[YakimaWA.kml.zip](https://github.com/ncss-tech/soilDB/files/13928209/YakimaWA.kml.zip)

Questions:

(1) A lot of WA676 data disappeared in the FY2024 dataset. Why? The area that disappears (bright color in map below) is now all `mukey` 2807830, which has the `muname` "Not public information". Did a UFO land here?
![image](https://github.com/ncss-tech/soilDB/assets/11711566/4b29d493-bbdd-4df4-bf59-40fc45bc3bec)

(2) This brightly color area in the FY2023 data is present, but it has STATSGO-esque, large mapunits. Why? Survey area WA676 comes up as "complete" in the "sastatusmap" table. 
```
soilDB::SDA_query("SELECT areasymbol, sapubstatuscode, iscomplete FROM sastatusmap WHERE areasymbol LIKE 'WA676'")
  areasymbol sapubstatuscode iscomplete
1      WA676               2          1
```

(3) What the heck is going on with that checkerboard pattern!? Now that definitely looks like a UFO-related phenomenon! Those areas are the only areas that persist within this survey area between FY23 and FY24, and they are also the only areas in this survey area with true, SSURGO-esque, high-resolution mapunits.

**Comment by dylanbeaudette** [comment, marked as answer]
I'll ask around, but this is probably tribal land and was requested the soil survey data be made "not public". The checkerboard pattern is probably a relic from the early days of the railroad.

https://en.wikipedia.org/wiki/Checkerboarding_(land)

**Comment by kevinwolz** [comment]
Well @#$^*&. That's something they don't teach you in school. I can't believe I haven't heard of that. _"The Dawes Act of 1887 created the most Native American checkerboarding. The act was intended to bolster self-sufficiency and systematically fracture native cultures."_ Insane.
